aot/src/test/resources/com/dylibso/chicory/approvals/ApprovalTest.verifyFloat.approved.txt

```diff
@@ -42,8 +42,8 @@ public final class com/dylibso/chicory/$gen/CompiledMachine implements com/dylib
     ARETURN
 
   public static func_0(Lcom/dylibso/chicory/runtime/Memory;Lcom/dylibso/chicory/runtime/Instance;)V
-    LDC 0.12345678
-    LDC 0.123456789012345
+    LDC 0.12345678F
+    LDC 0.123456789012345D
```

Comment:
Not really a fix since this is simply debug output, but the previous output was ambiguous with regard to the constant type, so this is definitely an improvement.

Reply:
I found the change here: https://gitlab.ow2.org/asm/asm/-/merge_requests/402